### PR TITLE
Cleanup lib versions 

### PR DESCRIPTION
**Pull request by oysteoh**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -12,7 +12,8 @@ sphinx
 cwrap
 numpy
 pandas
-matplotlib<3
+matplotlib<3; python_version < '3.0'
```
Comment (markusdregi) [inline requirements.txt]: What would happen if we just said `matplotlib` instead? As far as I know matplotlib only supports Python 2 in its `2.x` versions. Hence, you should automatically get `matplotlib<3` if you are in a Python 2 environment. Unless your environment is broken from the beginning, but I guess that is beyond the scope of a poor requirements-file?

Comment (oysteoh) [reply requirements.txt]: I don't know - it was you which added this <3 from poking gitblame...  I guess we could try?

Comment (markusdregi) [reply requirements.txt]: All the blame... I have no idea why I did that. Let us remove it and see how it goes

Comment (oysteoh) [reply requirements.txt]: ok, then i'll do that :)

File: requirements.txt [hunk 2]
```diff
@@ -11,8 +11,8 @@
 sphinx
 cwrap
 numpy
-pandas
-matplotlib<3
+pandas<0.25 # Pandas > 0.25 is not possible on RHEL6 since it requires some libs to be provided by the os
```
Comment (markusdregi) [inline requirements.txt]: But _ERT_ targets other platforms then RHEL 6. Is this a restriction of _ERT_ or of RHEL 6? If it is the first then the bound should stay, if it is the later this is the responsibility of the deployer.

Comment (oysteoh) [reply requirements.txt]: This was implemented pre our discussion with regards to komodo reinstalling libs and its deps and by that bumped pandas through 0.25 and broke our environment. I will remove this commit, and make a new issue in komodo adressing this which should be solved with --no-deps and with us defining all versions of all deps.

**Pull request merged**